src/Tizen.Applications.AttachPanel/Tizen.Applications.AttachPanel/AttachPanelInternal.cs

```diff
@@ -10,7 +10,7 @@ namespace Tizen.Applications.AttachPanel
     /// <since_tizen> 4 </since_tizen>
     public partial class AttachPanel
     {
-        private static IntPtr s_attachPanel = IntPtr.Zero;
+        private IntPtr s_attachPanel = IntPtr.Zero;
 
         private static event EventHandler<StateEventArgs> s_eventEventHandler;
```

Comment:
Would you remove static keyword for s_eventEventHanlder and s_resultEventHandler?

Reply:
Also, please change names of variables.
s_attachPanel => _attachPanel
s_eventEventHandler => _eventEventHandler
s_resultEventHandler => _resultEventHandler